pyls/plugins/flake8_lint.py

```diff
@@ -64,13 +74,15 @@ def build_args(options, doc_path):
     """
     args = [doc_path]
     for arg_name, arg_val in options.items():
+        if arg_val is None:
+            continue
         arg = None
         if isinstance(arg_val, list):
             arg = '--{}={}'.format(arg_name, ','.join(arg_val))
         elif isinstance(arg_val, bool):
             if arg_val:
                 arg = '--{}'.format(arg_name)
-        elif isinstance(arg_val, int):
+        else:
             arg = '--{}={}'.format(arg_name, arg_val)
         if arg:
```

Comment:
I think we should get rid of this check as you made sure `arg` always holds a value

Reply:
updated